ci-operator/step-registry/cucushift/installer/rehearse/azure/aro/vnet/cucushift-installer-rehearse-azure-aro-vnet-ref.metadata.json

```diff
@@ -0,0 +1,19 @@
+{
+	"path": "cucushift/installer/rehearse/azure/aro/vnet/cucushift-installer-rehearse-azure-aro-vnet-ref.yaml",
```

Comment:
1. Is it possible aro vnet re-use the `azure-provision-vnet` step as common OCP install?
2. if no, probably we need to place the step in `ci-operator/step-registry` folder (e.g: `ci-operator/step-registry/azure/provision/aro-net`), `cucushift/installer/rehearse` folder is for saving cluster install workflow yaml files.

Reply:
The aro vnet required is slightly different than the standard Azure one, it probably makes sense to give it its own step.

We have ci-operator/step-registry/aro/  which got checked in by mistake in a previous PR, I will see if I can move all the steps there and get it working again.

Reply:
vnet provioner now lives in ci-operator/step-registry/aro/provision/vnet